docs/guides/controller-autoconfiguration.md

```diff
@@ -19,9 +19,9 @@ We compute a matching score for each configuration file based on these three fac
 
 RetroArch works many platforms. Each of these platforms has one or more input systems. These input systems in turn differ widely in the way they enumerate the pad buttons. For this reason, your joypad buttons may be mapped differently depending on if you are using Windows, Mac, or Linux.
 
-Traditional emulators allow you map each button of your pad to the original pad of the emulated system. For example, this is how the Snes9x joypad configuration interface looks:
+Traditional emulators allow you map each button of your pad to the original pad of the emulated system. For example, this is how the BSNES joypad configuration interface looks:
 
-![Snes9x joypad configuration](../image/retroarch/input/snes9x-joyconfig-example.jpg)
+![image](https://github.com/davidhedlund/docs/assets/6416073/d89aefdb-ea5e-40be-8443-d34d88766d25)
```

Comment:
Reference local images, rather than 404s.

Reply:
Sorry. How can I upload the new image then?

Reply:
> Sorry. How can I upload the new image then?

You can put it into ../image/ folder. Make sure you put it into the correct sub-directory.

Reply:
> You can put it into ../image/ folder.

How?

Reply:
> > Sorry. How can I upload the new image then?
> 
> You can put it into ../image/ folder. Make sure you put it into the correct sub-directory.

Check this please: https://github.com/libretro/docs/compare/master...davidhedlund:docs:master

Reply:
I removed the image from the document, I don't have time for this right now.